WordPress/Sniffs/VIP/DirectDatabaseQuerySniff.php

```diff
@@ -167,7 +167,8 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 		}
 
 		$whitelisted_cache = false;
-		$cached            = $wp_cache_get = false;
+		$cached            = false;
+		$wp_cache_get      = false;
```

Comment:
I think there could also be an aspect of semantics here too - the last pointer IS the end of the line, where as the cached _thing_ is not the same as the WP cache get _thing_. If so, then the current code satisfies that, so all good.